pandas/tests/series/methods/test_describe.py

```diff
@@ -149,3 +150,25 @@ def test_datetime_is_numeric_includes_datetime(self):
             index=["count", "mean", "min", "25%", "50%", "75%", "max"],
         )
         tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "dtype", ["int32", "int64", "uint32", "uint64", "float32", "float64"]
+    )
+    def test_numeric_result_is_float(self, dtype):
```

Comment:
Maybe ``any_real_numpy_dtype``? Not sure if this fits the requirement here of what you want to test

Reply:
Thanks - I also tried complex and found this was buggy in that case. Fixed and added all numeric dtypes.